proto/sequencerapis/astria/sequencer/v1alpha1/block.proto

```diff
@@ -41,3 +42,19 @@ message SequencerBlock {
   // the rollup transactions.
   astria.sequencer.v1alpha1.Proof rollup_ids_proof = 4;
 }
+
+// `Deposit` represents a deposit from the sequencer
```

Comment:
Maybe?
```suggestion
// `Deposit` is a transfer of funds from the sequencer to a rollup.
```
Also, in which context is this used? Can you explain why this is not an action? From a first glance it looks very action-y.

Reply:
it's more similar to an "event", in that the sequencer application detects deposits and their corresponding rollup IDs so that the conductors/rollups don't have parse all the bridge lock txs in a block. a `Deposit` isn't exactly a transfer of funds (since that's implemented by the rollup), it's just notifying a rollup that funds were locked to some account on the sequencer. it's up to the rollup what to do with that info

Reply:
will add this to the comment